src/content/learn/scaling-up-with-reducer-and-context.md

```diff
@@ -219,7 +219,7 @@ For example, `TaskApp` passes a list of tasks and the event handlers to `TaskLis
 />
 ```
 
-And `TaskList` passes the event handlers to `Task`:
+Dan `TaskList` meneruskan *event handlers* ke `Task`:
```

Comment:
Let's use `mengoper` because other merged PRs already use this to translate `pass`.

```suggestion
Dan `TaskList` mengoper *event handlers* ke `Task`:
```